cmd/build/helmify/main.go

```diff
@@ -132,19 +132,17 @@ func (ks *kindSet) Write() error {
 			}
 
 			if name == "gatekeeper-controller-manager" && kind == DeploymentKind {
+				obj = strings.Replace(obj, "      labels:", "      labels:\n        {{- include \"controllerManager.podLabels\" . | nindent 8 }}\n        {{- include \"gatekeeper.commonLabels\" . | nindent 8 }}", 1)
 				obj = strings.Replace(obj, "      priorityClassName: system-cluster-critical", "      {{- if .Values.controllerManager.priorityClassName }}\n      priorityClassName:  {{ .Values.controllerManager.priorityClassName }}\n      {{- end }}", 1)
 			}
 
 			if name == "gatekeeper-audit" && kind == DeploymentKind {
 				obj = "{{- if not .Values.disableAudit }}\n" + obj + "{{- end }}\n"
+				obj = strings.Replace(obj, "      labels:", "      labels:\n        {{- include \"audit.podLabels\" . | nindent 8 }}\n        {{- include \"gatekeeper.commonLabels\" . | nindent 8 }}", 1)
 				obj = strings.Replace(obj, "      priorityClassName: system-cluster-critical", "      {{- if .Values.audit.priorityClassName }}\n      priorityClassName:  {{ .Values.audit.priorityClassName }}\n      {{- end }}", 1)
 				obj = strings.Replace(obj, "      - emptyDir: {}", "      {{- if .Values.audit.writeToRAMDisk }}\n      - emptyDir:\n          medium: Memory\n      {{ else }}\n      - emptyDir: {}\n      {{- end }}", 1)
 			}
 
-			if kind == DeploymentKind {
```

Comment:
Thanks for the PR @bobertrublik! We need to ensure backward compatibilities as to not break existing things users might be using and reduce developer toil. can you pls bring back all the existing podLabels and add support for new audit and controller manager podLabels?

Reply:
Of course, I'm just a little unsure how to approach this. Are you saying that I should readd the `.Values.podLabels` field to ensure backward compatibility? Are the labels set there added to all pods and we can additionally set the `.Values.controllerManager.podLabels` and `.Values.audit.podLabels` to add specific ones?

Reply:
yes, Pod can have many labels. Please restore the existing values.podLabels field to ensure backward compatibility. Then add the new labels.

Reply:
I restored `.values.podLabels` and kept `.Values.controllerManager.podLabels` and `.Values.audit.podLabels`. 🙂